Packs/ThreatIntelligenceManagement/Scripts/ThreatIntelligenceManagementGetIncidentsPerFeed/ThreatIntelligenceManagementGetIncidentsPerFeed.py

```diff
@@ -0,0 +1,115 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def get_default_from_date(date_range: str) -> str:
+    """
+    Gets a range string (eg. 30 days) and return a date string in the relevant Demisto query format.
+    :param date_range: string
+        Range (eg. 2 months) to create the date string from
+    :return: string
+        Date string in the relevant Demisto query format.
+    """
+    from_date, _ = parse_date_range(date_range=date_range)
+    str_from_date = from_date.strftime('%Y-%m-%dT%H:%M:%SZ')
+    return str_from_date
+
+
+def get_all_incidents(from_date: str) -> list:
+    """
+    Query all incidents starting a given date.
+    :param from_date: string
+        The date to query incidents from.
+    :return: list
+        List of incidents.
+    """
+    command_res = demisto.executeCommand('getIncidents', {'fromdate': from_date})
+    if is_error(command_res):
+        LOG(f'Error executing "getIncidents fromdate: {from_date}"')
```

Comment:
why not just `return_error(command_res)` ?

Reply:
Definitely forgot it also does LOG.